Modules/Microsoft365DSC/Modules/M365DSCReport.psm1

```diff
@@ -10,10 +10,6 @@ function New-M365DSCConfigurationToHTML
     [CmdletBinding()]
     [OutputType([System.String])]
     Param(
-        [Parameter()]
-        [System.String]
-        $ConfigurationPath,
```

Comment:
Why are we removing the ConfigurationPath here?

Reply:
See comment below.